README.md

```diff
@@ -172,7 +172,101 @@ when they expire.
 **Important**: ``credentials.state`` changes after a token swap. Be sure to persist
 the new state.
 
-
+### Using OAuth2 Credentials
+
+OAuth2 Credentials work similarly to the 3-step OAuth used by
+Partner Applications. OAuth2 tokens have a 30 minute expiry, 
+but can be swapped for a new token at any time. Procedure for authenticating
+credentials:
+
```

Comment:
There's a step 0 here - [creating the app](https://developer.xero.com/myapps), which will create the client_id and client_secret.

Reply:
I've addressed this in the documentation update.